### PR TITLE
fix(dark-theme): adjust Dark Reader CSS selectors

### PR DESCRIPTION
**Pull request by Arthurk12**
### What does this PR do?
Clean up inverted css selectors passed to Dark Reader and add new ones for elements not correctly transformed to dark theme. These include the tldraw color picker, text shape color, selected color indicator, tool opacity slider, and camera dock background.
Comparsion:
Before and after:
whiteboard_ whiteboard
cameraDock-background_
cameraDock-background

### CODE REVIEW

**Review by gabriellpr — APPROVED** [review]

File: bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx [hunk 1]
```diff
@@ -445,6 +446,13 @@ const Whiteboard = React.memo(function Whiteboard(props) {
     setTldrawAPI(editor);
 
     editor?.user?.updateUserPreferences({ locale: language });
+    
+    if (darkTheme) {
+      // force darkReader to reload to fix some cases
+      // where tldraw dark theme isn't applied
+      AppService.setDarkTheme(false);
+      AppService.setDarkTheme(true);
+    }
```
Comment (germanocaumo) [inline bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx:455]: From my tests this didn't seem to make a difference, did you notice it not applying sometimes?

File: bigbluebutton-html5/imports/ui/components/app/styles.js [hunk 2]
```diff
@@ -49,54 +52,41 @@ const DtfInvert = `
...
-  svg
+  svg,
+  g,
+  line,
+  textarea,
+  rect,
+  circle,
+  .tlui-buttons__grid > button,
+  .tlui-popover > button,
+  .tl-html-container > div.tl-text-shape__wrapper.tl-text-shadow
```
Comment (germanocaumo) [inline bigbluebutton-html5/imports/ui/components/app/styles.js:89]: More selectors to add to fix notes and text colors inside shapes (mainly arrows)
```suggestion
  .tl-html-container > div.tl-text-shape__wrapper.tl-text-shadow,
  .tl-text,
  .tl-text-input,
  .tl-text-content,
  .tl-text-label__inner,
  .tl-note__container,
  .tl-text.tl-text-content,
  .tl-arrow-label,
  .tl-arrow-label__inner
```

File: bigbluebutton-html5/imports/ui/components/app/styles.js [hunk 3]
```diff
@@ -49,54 +52,41 @@ const DtfInvert = `
...
-  }
-  #TD-Tools button, #TD-TopPanel-Undo, #TD-TopPanel-Redo, #TD-Styles {
-    border-color: transparent !important;
+  .bnjzQC > div span div:hover {
+    background-color: var(--darkreader-selection-background) !important;
   }
-  [id="TD-StylesMenu"],
-  [id="TD-Styles-Color-Container"],
-  div[data-test="brandingArea"],
   #connectionBars > div
```
Comment (germanocaumo) [inline bigbluebutton-html5/imports/ui/components/app/styles.js:70]: Some fixes to note background color and caret color
```suggestion
  #connectionBars > div,
  .tl-note__scrim,.tl-arrow-label[data-isediting="true"] > .tl-arrow-label__inner {
    background-color: unset !important;
  }
  textarea {
    caret-color: black !important;
  }
```

File: bigbluebutton-html5/imports/ui/components/app/styles.js [hunk 4]
```diff
@@ -49,54 +52,41 @@ const DtfInvert = `
   .tl-container {
     background-color: var(--tl-background) !important;
   }
```
Comment (germanocaumo) [inline bigbluebutton-html5/imports/ui/components/app/styles.js:54]: This is needed to fix background color in pdfs with transparent background 
```suggestion
  .tl-container {
    .tl-image {
      background-color: white !important;
    }
  }
```

**Review by germanocaumo — CHANGES_REQUESTED** [review]
Found some wrong colors, suggested changes to fix

File: bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx (hunk 1 above)
Comment (germanocaumo) [reply bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx:455]: @Arthurk12 yeah I think it's better to remove it, not needed in this tldraw version

**Review by germanocaumo — CHANGES_REQUESTED** [review]

**Review by germanocaumo — APPROVED** [review]

**Pull request merged**